src/AutoRest.CSharp/Common/Generation/Writers/RestClientWriter.cs

```diff
@@ -84,7 +88,7 @@ private static void WriteClientCtor(CodeWriter writer, DataPlaneRestClient restC
             writer.Line();
         }
 
-        private static void WriteOperation(CodeWriter writer, DataPlaneRestClient restClient, RestClientMethod operation, bool async)
+        public static void WriteOperation(CodeWriter writer, RestClientMethod operation, string pipelineName, bool async, WritePageFuncBody writePageFuncBody, string methodNameSuffix = "", WriteStatusCodeImplementation? writeStatusCodeImplementation = null, FieldDeclaration? fieldDeclaration = null)
```

Comment:
Why do we need 'methodNameSuffix'?  It looks the value is always empty.

Reply:
In HLC, the paging method and its firstpage method have the same name. They are in different classes (one in client and the other in restclient), so there is no issue. 

However, in DPG, they are in the same class, so we need to rename the firstpage method.